test/system/src/test/java/io/pravega/test/system/AbstractFailoverTests.java

```diff
@@ -179,6 +179,7 @@ private void closeWriter(EventStreamWriter<Long> writer) {
             writer.close();
         } catch (Throwable e) {
             log.error("Error while closing writer", e);
+            Assert.fail("Error while closing writer. Test failure");
```

Comment:
I think you need to do
`testState.getWriteException.set(e);` because this may be in another thread.

Reply:
done.